src/app/modules/timesheet/components/timesheet-edit/timesheet-edit.component.scss

```diff
@@ -1,244 +1,210 @@
-@import 'src/app/style/vars';
-@import 'src/app/style/mixins';
+@import "src/app/style/vars";
+@import "src/app/style/mixins";
 
-.ng-invalid:not(form)  {
+.ng-invalid:not(form) {
   border-left: 5px solid #a94442; /* red */
 }
 
 .edit-cra {
-    background: $grey_light;
+  &__wrapper {
+    @include container;
 
-    &__header {
-      @media (min-width: $screen-sm-min) {
-          height: 80px;
-      }
-      background: $main_blue;
-      padding: 10px 0;
-      height: 60px;
-      box-shadow: 0 0 8px rgba($black, .8);
-
-      &__container {
-        position: relative;
-        height: 100%;
-
-        a {
-          opacity: 1;
-          transition: opacity .15s ease-in-out;
-        }
+    @media (min-width: $screen-sm-min) {
+      padding: 40px;
+    }
 
-        a:hover {
-          opacity: 0.9;
-        }
+    @media (min-width: $screen-md-min) {
+      margin-top: 30px;
+    }
 
-        &__logo {
-          display: block;
-          position: absolute;
-                  left: 20px;
-          height: 100%;
+    margin-top: 10px;
+    background: $white;
+    padding: 20px;
+    box-shadow: 0 0 8px rgba($black, 0.2);
+
+    &__title {
+      font-size: 22px;
+      color: $grey_mine_shaft;
+
+      & > h2 {
+        @media (min-width: $screen-sm-min) {
+          font-size: 28px;
         }
-		  }
-	  }
 
-    &__wrapper {
-      @include container;
+        font-weight: 700;
+        font-size: 20px;
+      }
+    }
 
+    // &__modals {
```

Comment:
If you don't need this css you can't remove it ? In case of extreme necessity you can find it in github